React/Base/macOS/RCTUIKit.m

```diff
@@ -751,4 +751,93 @@ - (void)setHidden:(BOOL)hidden
 
 @end
 
+// RCTUIImageView
+
+@implementation RCTUIImageView
+
+@synthesize tintColor = _tintColor;
+@synthesize contentMode = _contentMode;
+
+- (instancetype)initWithFrame:(CGRect)frame
+{
+  if (self = [super initWithFrame:frame]) {
+    self.layer = [[CALayer alloc] init];
+    self.wantsLayer = YES;
```

Comment:
Our style guide is to avoid dot notation and use the setters. React Native is weird since we inherit a bunch of Meta code. However this file is macOS only... So I think we should follow the Microsoft one?

https://microsoft.github.io/objc-guide/Syntax/DotNotation.html

Reply:
Similar comments elsewhere